### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]

File: doc/articles/get-started-next-steps.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Already [completed the tutorial](getting-started-tutorial-1.md)? Ready to build your own app or library, or just want to learn more? These resources will help you on your way.
 
+* The Uno Platform brings Microsoft's UWP/WinUI framework to Android, iOS, macOS, Linux, and WebAssembly. The wealth of [official Microsoft documentation on the UWP framework](https://docs.microsoft.com/en-us/windows/uwp/develop/) is a great place to start, particularly if you've never used UWP/WinUI or Xaml before. Some additonal resources :
+   - [Get started with Windows 10 apps](https://docs.microsoft.com/en-us/windows/uwp/get-started/)
+   - [Universal Windows Platform documentation](https://docs.microsoft.com/en-us/windows/uwp/develop)
+   - [Udemy Course](https://www.udemy.com/course/introduction-to-uno-platform/?referralCode=C9FE308096EADFB5B661)
```
Comment (davidjohnoliver) [inline doc/articles/get-started-next-steps.md:8]: Udemy course deserves to be called out in a new paragraph

File: doc/articles/get-started-with-linux.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Now let's run the application:
 - Install GTK3:
     ```
     sudo apt update
-    sudo apt install gtk+3.0
+    sudo apt install libgtk-3-dev
```
Comment (davidjohnoliver) [inline doc/articles/get-started-with-linux.md:70]: 👍

File: doc/articles/get-started-wizard.md [hunk 3]
```diff
@@ -13,30 +13,53 @@ Congratulations, you've just created a new project using the [Uno Platform](http
...
 
-To work around this issue, close all XAML editors, open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
+To work around this issue:
+- Close all XAML editors
+- If you just created the project in the same Visual Studio 2019 instance:
+  - Close the solution and/or restart Visual Studio
+  - Reopen the solution
+- Open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
+
+> This issue has been fixed by Microsoft in VS 16.9 Preview 3.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:32]: 🎉

File: doc/articles/uno-builds-troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+In order to clear the **Error List** window, build the whole solution completely once.
+Subsequently, build a specific project and prefer the use of the **Output** tool window (in the menu **View** -> **Output**), taking build messages by order of appearance.
+
+## Troubleshooting build errors using the build Output Window
+To troubleshoot build error, you can change the text output log level:
+  - Go to **Tools**, **Options**, **Projects and Solution**, then **Build and Run**
+  - Set **MSBuild project build output verbosity** to **Normal** or **Detailed**
+  - Build your project again and take a look at the additional output
+
+## Generating msbuild Binary Log files
```
Comment (davidjohnoliver) [inline doc/articles/uno-builds-troubleshooting.md:14]:
```suggestion
## Generating MSBuild Binary Log files
```

**Review by davidjohnoliver — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**